js/App.js

```diff
@@ -18,7 +18,10 @@ import Home from './scenes/Home';
 import Item from './scenes/Item';
 import Map from './scenes/Map';
 
-export default class App extends Component {
+import { connect } from 'react-redux';
+import { watchLocation } from './store/actions';
+
+class App extends Component {
```

Comment:
We want to export this class too for testing purpose. See other `connect`ed classes.